### ISSUE TITLE
Javascript not loaded in newest dockerfile

### ISSUE THREAD

**Issue by hroest** [bgruening/docker-galaxy-ngs-preprocessing#18]
There is some error with access to js libraries in the newest version:

```
$  docker run -i -t -p 8081:80 bgruening/galaxy-ngs-preprocessing
```

![screenshot from 2018-09-22 12-28-29](https://user-images.githubusercontent.com/464014/45919446-236a2a00-be63-11e8-8627-a521e6cc8b91.png)

while running the older version works without issue

```
$ docker run -i -t -p 8080:80 bgruening/galaxy-ngs-preprocessing:18.01 # this works
```

**Comment by bgruening** [comment]
Hannes can you try the quay.io version? Dockerhub is unreliable in its builds recently.

**Comment by hroest** [comment]
great, that works

```
docker run -i -t -p 8081:80 quay.io/bgruening/galaxy-ngs-preprocessing
```

maybe then we should adjust the README ? / and delete the broken dockerhub build?

**Comment by bgruening** [comment]
Yeah you are probably right. I had hope that the dockerhub failures are temporary.... :-(

**Comment by hroest** [comment]
its just sub-optimal if the latest build is a broken version - do you know why dockerhub is broken?

**Comment by bgruening** [comment]
It's not broken, it's just super unreliable. It's not building everytime it should, builds are not finishing and so on ...
